templates/cisco_xr_show_bgp_neighbors.template

```diff
@@ -23,6 +24,7 @@ Value List PREFIXES_OUT_SUPPRESSED (\d+)
 Value List PREFIXES_OUT_WITHDRAWN (\d+)
 Value List MAX_PREFIXES (\d+)
 Value List MAX_PREFIXES_WARN_PERCENT (\d+)
+Value List DEFAULT_ORIGINATE (.+)
```

Comment:
```suggestion
Value List DEFAULT_ORIGINATE (.+?)
```

Reply:
so with ```  Default information originate: default sent``` we want to match to the end of the line including whitespace or else it will only capture "default" instead of "default send", no?

Reply:
It will capture everything up to the end of the line excluding a space. If the line doesn't have a space, it will still capture all of the text (that is why we use `*` instead of `+`)